docker/package/baking_wizard.py

```diff
@@ -0,0 +1,303 @@
+            validator=required_field_validator,
+            reqs=lambda vs: vs["mode"] == "baker" and vs["key_import_mode"] == "ledger",
+        ),
+        Step(
+            id="secret_key",
+            prompt="Provide either the unencrypted or password-encrypted secret key for your address.\n",
+            help="The format is 'unencrypted:edsk...' for the unencrypted key, or 'encrypted:edesk...'"
+            "for the encrypted key.",
+            validator=required_field_validator,
+            reqs=lambda vs: vs["mode"] == "baker"
+            and vs["key_import_mode"] == "secret-key",
+        ),
+        Step(
+            id="json_filepath",
+            prompt="Provide the path to your downloaded faucet JSON file.\n",
+            help="The JSON file will contain the 'mnemonic' and 'secret' fields.",
+            validator=[required_field_validator, filepath_validator],
+            reqs=lambda vs: vs["mode"] == "baker" and vs["key_import_mode"] == "json",
+        ),
+    ]
+
+    for step in steps:
+        if not step.reqs(values):
+            continue
+
+        readline.set_completer(step.completer)
+
+        validated = False
+        while not validated:
+            try:
+                answer = input(step.prompt).strip()
+            except:
+                print("Exiting Tezos Baking Wizard")
+                return None
+
+            if answer.lower() in ["quit", "exit"]:
+                print("Exiting Tezos Baking Wizard")
+                return None
+            elif answer.lower() == "help":
+                print(step.help)
+            else:
+                if not answer and step.default is not None:
+                    answer = step.default
+
+                try:
+                    step.validate(answer)
+                except ValueError as e:
+                    print("Validation error: ", str(e))
+                else:
+                    validated = True
+                    values[step.id] = answer
+
+    return values
+
+
+class Setup:
+    def __init__(self, config):
+        self.config = config
+
+    # Bootstrapping tezos-node
+    def bootstrap_node(self):
+        if self.config["snapshot"]:
+            proc_call(
+                "sudo -u tezos tezos-node-"
+                + self.config["network"]
+                + " snapshot import "
+                + self.config["snapshot"]
+            )
+
+        proc_call(
+            "sudo systemctl start tezos-node-" + self.config["network"] + ".service"
+        )
+
+        return_code = None
+        while return_code != 0:
+            os.system("sleep 5")
+            return_code = subprocess.call(
+                shlex.split("sudo -u tezos tezos-client bootstrapped")
+            )
+
+    # Importing the baker key
+    def import_baker_key(self):
+        if self.config["key_import_mode"] == "secret-key":
+            proc_call(
+                "sudo -u tezos tezos-client import secret key baker"
```

Comment:
```suggestion
                "sudo -u tezos tezos-client import secret key baker "
```
looks like space is missing, the same for the rest of import invocations